Example/Krypt.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -4,5 +4,7 @@
 <dict>
 	<key>BuildSystemType</key>
 	<string>Original</string>
+	<key>PreviewsEnabled</key>
```

Comment:
Is this for SwiftUI?

Reply:
Yes. It was added by Xcode automatically. Should I remove it?

> Xcode 11 displays previews of your user interface right in the editor, streamlining the edit-debug-run cycle into a seamless workflow. Optimize the structure of your SwiftUI app for previews, and how to add preview support to your existing views and view controllers.

Reply:
I'm just wondering why it's `false`